scripts/table_updates/config.json

```diff
@@ -18,10 +18,14 @@
         "BLADDER": "syn26721151",
         "NSCLC2": "syn51318736",
         "CRC2": "syn52943210",
-        "RENAL": "syn59474249"
+        "RENAL": "syn59474249",
         "OVARIAN": "syn64042773"
     },
-    "main_genie_release_version": "16.6-consortium",
+    "tier1a_replacement_mapping":{
+        "patient_characteristics_tier1a_replacement_mapping_table": "syn64331052",
```

Comment:
May I ask what these tables are for?

Reply:
The `patient_characteristics_tier1a_replacement_mapping_table` and `cancer_panel_test_tier1a_replacement_mapping_table` tracks how original bpc codes are mapped to NAACCR code. This is added to address Chelsea's request: "adds to a log if there is a mismatch between the upload and the replaced value".